pkg/gorgias/config.go

```diff
@@ -0,0 +1,15 @@
+package gorgias
+
+import (
+	"fmt"
+)
+
+type Config struct {
+	Domain string
+	APIKey string
+	Email  string
+}
+
+func (c *Config) GetIngestrURI() string {
+	return fmt.Sprintf("gorgias://%s?api_key=%s&%s", c.Domain, c.APIKey, c.Email)
```

Comment:
seems like we are missing the key for the email:

```suggestion
	return fmt.Sprintf("gorgias://%s?api_key=%s&email=%s", c.Domain, c.APIKey, c.Email)
```